src/llvm_interpreter.rs

```diff
@@ -702,6 +702,8 @@ struct LLVMBoxedStructRepr {
 }
 
 impl LLVMJIT {
+    //FIXME: not sure why this is seen as a "useless" transmute, I need to cast it to a void*
+    #[cfg_attr(feature = "cargo-clippy", allow(useless_transmute))]
```

Comment:
What clippy suggests is: `libc::malloc as usize as *mut libc::c_void` which seems weird, but I guess makes sense.

Also, you can actually use this attribute on only the actual function call by placing it on the line before the `mem::transmute`.